bundler/spec/dependabot/bundler/file_updater/gemfile_updater_spec.rb

```diff
@@ -299,7 +305,11 @@
           }]
         end
 
-        it { is_expected.to include "\"business\", \"~> 1.8.0\"\n" }
+        it do
+          is_expected.to include(
+            "\"dependabot-test-ruby-package\", \"~> 1.0.0\", git"
```

Comment:
Hm looks like this test has changed, looks like it was testing whether the git source was removed from the lockfile if the new requirement was a published version where the previous_requirements was a git source. Might need to publish this package with a newer version to test it?

Reply:
I messed up the versions and missed that all these tests were nested inside the same block, it should be fixed now